src/main/java/org/tndata/android/compass/activity/ChooseGoalsActivity.java

```diff
@@ -321,11 +321,13 @@ public void goalSelected(Goal goal) {
             //mSelectedGoals.add(goal);
             addGoal(goal);
 
-            // Launch the GoalTryActivity (where users choose a behavior for the Goal)
-            Intent intent = new Intent(getApplicationContext(), GoalTryActivity.class);
-            intent.putExtra("goal", goal);
-            intent.putExtra("category", mCategory);
-            startActivity(intent);
+            if (goal.getBehaviorCount() > 0){
+                // Launch the GoalTryActivity (where users choose a behavior for the Goal)
+                Intent intent = new Intent(getApplicationContext(), GoalTryActivity.class);
+                intent.putExtra("goal", goal);
+                intent.putExtra("category", mCategory);
+                startActivity(intent);
+            }
```

Comment:
Let's add an `else` block with a Toast that includes something like "Goal Added". We do that in other places, so let's be consistent.